templates/generic_show_form.html.twig

```diff
@@ -645,6 +645,11 @@
                         {{ fields.field('is_dynamic', is_dynamic_value, __('Automatic Inventory')) }}
                      {% endif %}
 
+                     {# display last_boot data only if item is dynamic and have field #}
+                     {% if item.isField('last_boot') and item.isField('is_dynamic') and item.fields['is_dynamic'] %}
+                        {{ fields.htmlField('last_boot', item.fields['last_boot'], __('Last boot')) }}
```

Comment:
To be tested, but I think it will make it more readable.
```suggestion
                        {{ fields.htmlField('last_boot', item.fields['last_boot']|formatted_datetime, __('Last boot')) }}
```

Reply:
before : 
![image](https://user-images.githubusercontent.com/7335054/194517525-49e73b22-5a57-4ffe-9eeb-becf03ebc950.png)

after : 
![image](https://user-images.githubusercontent.com/7335054/194517554-c59471f5-6e35-472f-827a-8ac1539ee9d3.png)

```seconds```re lost